script/test-versions

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+# This should be run inside a container built from the Dockerfile
+# at the root of the repo - script/test will do it automatically.
+
+set -e
+
+>&2 echo "Running lint checks"
+flake8 compose
+
+if [ "$DOCKER_VERSIONS" == "" ]; then
+  DOCKER_VERSIONS="1.5.0"
+elif [ "$DOCKER_VERSIONS" == "all" ]; then
+  DOCKER_VERSIONS="1.3.3 1.4.1 1.5.0"
```

Comment:
If you set this in the `Dockerfile` instead, you could have the entire list of "supported" Docker versions in a single place: (in case that's an interesting property for you)
```Dockerfile
ENV SUPPORTED_DOCKER_VERSIONS 1.5.0 1.4.1 1.3.3
RUN set -e \
    && for v in ${SUPPORTED_DOCKER_VERSIONS}; do \
        curl https://get.docker.com/builds/Linux/x86_64/docker-$v -o /usr/local/bin/docker-$v; \
        chmod +x /usr/local/bin/docker-$v; \
    done
```